### ISSUE TITLE
Multiple progress bars

### ISSUE THREAD

**Issue by 0xF6** [Semantic-Org/Semantic-UI#6160]
It would be very good if there were multiple bars
Alt: [Here](https://getbootstrap.com/docs/4.0/components/progress/#multiple-bars)

**Comment by Steveb-p** [comment]
Not including JS, it can be achieved by switching `.ui.progress` display property to flex and changing rounding for children (not included in sample screenshot).

![image](https://user-images.githubusercontent.com/3183926/42688186-9859a886-869b-11e8-9be1-560a09eddb21.png)

I've checked other styles included in semantic at this time (classic, basic, default, striped) and they seem to behave properly.

**Comment by azmeuk** [comment]
@Steveb-p could you please provide a JSfiddle?

**Comment by Steveb-p** [comment]
@azmeuk Will do in the evening. I'll try to remember to do so ;)

**Comment by Hoohaha** [comment]
Have any update? I really hope semantic support this feature.

**Comment by azmeuk** [comment]
@Steveb-p Just a remember for a fiddle! :)

**Comment by azmeuk** [comment]
@Steveb-p :) are you still around there?

**Comment by Steveb-p** [comment]
@azmeuk I didn't really remember to post that did I? :D

I'll look for that code.

**Comment by Steveb-p** [comment]
@azmeuk https://jsfiddle.net/c3g5u1jz/

I haven't checked for any side-effects though. It really was a simple attempt.

**Comment by lubber-de** [comment]
Multiple Progress Bars are implement in Fomantic-UI 2.7.3 now 😄 
https://fomantic-ui.com/modules/progress.html#multiple-bars
![image](https://user-images.githubusercontent.com/18379884/55389068-d1524180-5534-11e9-9cac-9e949df72342.png)

**Comment by Steveb-p** [comment]
@lubber-de What is the relationship between semantic-ui & fomantic?

**Comment by lubber-de** [comment]
@Steveb-p 
TL;DR;
Fomantic keeps Semantic alive by providing long awaited bugfixes and enhancements.

From our https://github.com/fomantic/Fomantic-UI/blob/master/README.md

>Fomantic was created to continue active development of Semantic-UI and has the intent to be merged >back into the master repository once active development can restart. For more info please read the >following issues 
https://github.com/Semantic-Org/Semantic-UI/issues/6109
https://github.com/Semantic-Org/Semantic-UI/issues/6413

**Comment by Steveb-p** [comment]
@lubber-de Guess I'll switch my apps to Fomantic for the time being then :dancer:
